src/modules/commander/Commander.cpp

```diff
@@ -685,42 +739,28 @@ Commander::handle_command(const vehicle_command_s &cmd)
 					if (cmd.source_system == _status.system_id && cmd.source_component == _status.component_id
 					    && cmd_from_io && cmd_arms) {
 						_status.arming_state = vehicle_status_s::ARMING_STATE_IN_AIR_RESTORE;
```

Comment:
Could this in-air restore get overridden by the normal arming transition that follows afterwards because it's not in the else case anymore? (Does it work at all? Couldn't it be handled like a force arm?)

Reply:
I don't think this needs any action in this pr. Discussion about how to (not) support the feature in the future was kicked off.